test/test_genbank_sbol3_direct.py

```diff
@@ -150,6 +150,11 @@ def test_round_trip_multiple_loc_feat(self):
             Path(__file__).parent / "test_files" / "sbol3_genbank_conversion" / "multiple_feature_locations.gb"
         )
         self._test_round_trip_genbank(genbank_file)
+
+    def test_round_trip_all_testfiles(self):
+        test_file_dir = Path(__file__).parent / 'test_files'
+        for genbank_file in test_file_dir.glob('*.gb'):
+            self._test_round_trip_genbank(Path(genbank_file))
```

Comment:
Super sorry. This is quite minor. `pathlib.Path.glob` returns objects of type `pathlib.Path`, so you can just use `genbank_file` as is.

```suggestion
            self._test_round_trip_genbank(genbank_file)
```